src/lib/Components/Show/ShowArtworksPreview.tsx

```diff
@@ -21,7 +21,7 @@ export class ShowArtworksPreview extends React.Component<Props> {
     return (
       <>
         {title && (
-          <Serif size="6" mt={3} mb={3}>
+          <Serif size="6" mt={2} mb={3}>
```

Comment:
again super minor (and i see that yr updating other code), but better to wrap <Serif> / typography in <Box> and adjust margins there since box is more structural / layout-like